package javaguru.sunday.student_liana_shirmane.lesson_12.level_3;
//task20

//Exception 'java.lang.ArithmeticException' has already been caught by parent class Exception

public class MultiCatchExample {
    public static void main(String[] args) {
        try {

        } catch (Exception e) {

        }
    }
}
